core-build/gulp-core-build-serve/src/certificates.ts

```diff
@@ -85,7 +85,8 @@ function _ensureCertUtilExePath(parentTask: GulpTask<{}>): string {
       parentTask.logError(`Error finding certUtil command: "${whereErr}"`);
       _certutilExePath = undefined;
     } else {
-      _certutilExePath = where.stdout.toString().trim();
+      const lines: string[] = where.stdout.toString().trim().split(EOL);
+      _certutilExePath = lines[0].trim();
```

Comment:
There should only be one. If there are two (i.e. - one 32-bit and one 64-bit), either should work and we should just fail if one doesn't.